File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/bike/BikeDetailResponse.java
package com.lattis.ellipse.data.network.model.response.bike;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/13/17.
 */

public class BikeDetailResponse {

    @SerializedName("payload")
    BikeDetailPayloadResponse bikeDetailPayloadResponse;

    public FindBikeDataResponse getBikeDetailResponse() {
        return bikeDetailPayloadResponse.getBikeDetailDataResponse();
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/bike/BikeDetailResponse.kt
package com.lattis.data.entity.response.bike

import com.google.gson.annotations.SerializedName


class BikeDetailResponse {

    @SerializedName("payload")
    internal var bikeDetailPayloadResponse: BikeDetailPayloadResponse? = null

    val bikeDetailResponse: FindBikeDataResponse?
        get() = bikeDetailPayloadResponse!!.bikeDetailDataResponse
}
